regparser/tree/xml_parser/reg_text.py

```diff
@@ -160,8 +161,16 @@ def next_marker(xml_node, remaining_markers):
 def build_from_section(reg_part, section_xml):
     section_texts = []
     nodes = []
+
+    section_no = section_xml.xpath('SECTNO')[0].text
+    subject_xml = section_xml.xpath('SUBJECT')
+    if not subject_xml:
+        subject_xml = section_xml.xpath('RESERVED')
+    subject_text = subject_xml[0].text
+
     # Collect paragraph markers and section text (intro text for the
     # section)
+    i = 0
```

Comment:
I'm a fan of single letter variables, but in this case 'i' is not descriptive. In either case, it doesn't look like this i is incremented anywhere. Am I missing something?